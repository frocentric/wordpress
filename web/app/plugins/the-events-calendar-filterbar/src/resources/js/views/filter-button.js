/* global tribe */
/* eslint-disable no-var, strict */

/**
 * Makes sure we have all the required levels on the Tribe Object.
 *
 * @since 5.0.0
 *
 * @type   {PlainObject}
 */
tribe.filterBar = tribe.filterBar || {};

/**
 * Configures Filter Button Object in the Global Tribe variable.
 *
 * @since 5.0.0
 *
 * @type   {PlainObject}
 */
tribe.filterBar.filterButton = {};

/**
 * Initializes in a Strict env the code that manages the filter Button.
 *
 * @since 5.0.0
 *
 * @param  {PlainObject} $   jQuery
 * @param  {PlainObject} obj tribe.filterBar.filterButton
 *
 * @return {void}
 */
( function( $, obj ) {
	'use strict';
	var $document = $( document );

	/**
	 * Selectors used for configuration and setup.
	 *
	 * @since 5.0.0
	 *
	 * @type {PlainObject}
	 */
	obj.selectors = {
		actionDone: '[data-js="tribe-filter-bar__action-done"]',
		filterBarVertical: '.tribe-events--filter-bar-vertical',
		filterBar: '[data-js~="tribe-filter-bar"]',
		filterButton: '[data-js~="tribe-events-filter-button"]',
		filterButtonActive: '.tribe-events-c-events-bar__filter-button--active',
		filtersSliderContainer: '[data-js="tribe-filter-bar-filters-slider-container"]',
		select2ChoiceRemove: '.select2-selection__choice__remove',
	};

	/**
	 * Handler for resize event on vertical filter bar.
	 *
	 * @since  5.0.0
	 *
	 * @param  {Event} event event object of click event.
	 *
	 * @return {void}
	 */

	obj.handleResize = function( event ) {
		var $container = event.data.container;
		var $filterBar = $container.find( obj.selectors.filterBar );
		var state = $filterBar.data( 'tribeEventsState' );
		var containerState = $container.data( 'tribeEventsState' );
		var isMobile = containerState.isMobile;

		// Set filter bar state on resize to desktop.
		if ( ! isMobile && ! state.filterButtonDesktopInitialized ) {
			// Open vertical filter bar on resize to desktop.
			if ( $container.is( obj.selectors.filterBarVertical ) ) {
				tribe.filterBar.filterBarState.openFilterBar( $container );
			}

			state.filterButtonDesktopInitialized = true;
			$filterBar.data( 'tribeEventsState', state );

		// Close filter bar on resize to mobile.
		} else if ( isMobile && state.filterButtonDesktopInitialized ) {
			tribe.filterBar.filterBarState.closeFilterBar( $container );
			state.filterButtonDesktopInitialized = false;
			$filterBar.data( 'tribeEventsState', state );
		}
	};

	/**
	 * Handler for document click event.
	 *
	 * @since  5.0.0
	 *
	 * @param  {Event} event event object of click event.
	 *
	 * @return {void}
	 */
	obj.handleClick = function( event ) {
		var $container = event.data.container;
		var containerState = $container.data( 'tribeEventsState' );
		var isMobile = containerState.isMobile;

		// Return early if not mobile.
		if ( ! isMobile ) {
			return;
		}

		var $target = $( event.target );
		var isParentFilterBar = Boolean( $target.closest( obj.selectors.filterBar ).length );
		var isParentFilterButton = Boolean( $target.closest( obj.selectors.filterButton ).length );
		var isParentSelect2ChoiceRemove = Boolean( $target.closest( obj.selectors.select2ChoiceRemove ).length );

		if ( ! isParentFilterBar && ! isParentFilterButton && ! isParentSelect2ChoiceRemove ) {
			tribe.filterBar.filterBarState.closeFilterBar( $container );
		}
	};

	/**
	 * Handler for action done button click event.
	 *
	 * @since  5.0.0
	 *
	 * @param  {Event} event event object of click event.
	 *
	 * @return {void}
	 */
	obj.handleActionDoneClick = function( event ) {
		tribe.filterBar.filterBarState.closeFilterBar( event.data.container );
	};

	/**
	 * Handler for filter button click event.
	 *
	 * @since  5.0.0
	 *
	 * @param  {Event} event event object of click event.
	 *
	 * @return {void}
	 */
	obj.handleFilterButtonClick = function( event ) {
		var $container = event.data.container;
		var $filterButton = event.data.target;
		var isOpen = $filterButton.is( obj.selectors.filterButtonActive );

		if ( isOpen ) {
			tribe.filterBar.filterBarState.closeFilterBar( $container );
		} else {
			tribe.filterBar.filterBarState.openFilterBar( $container );
		}

		const data = { container: $container };
		tribe.filterBar.filterBarSlider.handleResize( { data: data } );
	};

	/**
	 * Unbind events for filter toggles functionality.
	 *
	 * @since  5.0.0
	 *
	 * @param  {jQuery} $container jQuery object of view container.
	 *
	 * @return {void}
	 */
	obj.unbindEvents = function( $container ) {
		var $filterButton = $container.find( obj.selectors.filterButton );
		var $actionDone = $container.find( obj.selectors.actionDone );
		$filterButton.off( 'click', obj.handleFilterButtonClick );
		$actionDone.off( 'click', obj.handleActionDoneClick );
		$document.off( 'click', obj.handleClick );
		$container.off( 'resize.tribeEvents', obj.handleResize );
	};

	/**
	 * Bind events for filter button functionality.
	 *
	 * @since  5.0.0
	 *
	 * @param  {jQuery} $container jQuery object of view container.
	 *
	 * @return {void}
	 */
	obj.bindEvents = function( $container ) {
		var $filterButton = $container.find( obj.selectors.filterButton );
		var $actionDone = $container.find( obj.selectors.actionDone );
		$filterButton.on(
			'click',
			{ target: $filterButton, actionDone: $actionDone, container: $container },
			obj.handleFilterButtonClick,
		);
		$actionDone.on(
			'click',
			{ target: $actionDone, filterButton: $filterButton, container: $container },
			obj.handleActionDoneClick,
		);
		$document.on( 'click', { container: $container }, obj.handleClick );
		$container.on( 'resize.tribeEvents', { container: $container }, obj.handleResize );
	};

	/**
	 * Initializes filter button state.
	 *
	 * @since  5.0.0
	 *
	 * @param  {jQuery} $container jQuery object of view container.
	 *
	 * @return {void}
	 */
	obj.initState = function( $container ) {
		var $filterBar = $container.find( obj.selectors.filterBar );
		var containerState = $container.data( 'tribeEventsState' );
		var isMobile = containerState.isMobile;

		var state = {
			filterButtonDesktopInitialized: ! isMobile,
		};

		$filterBar.data( 'tribeEventsState', state );
	};

	/**
	 * Deinitialize filter button JS.
	 *
	 * @since  5.0.0
	 *
	 * @param  {Event} event event object for 'beforeAjaxSuccess.tribeEvents' event.
	 *
	 * @return {void}
	 */
	obj.deinit = function( event ) {
		var $container = event.data.container;
		obj.unbindEvents( $container );
		$container.off( 'beforeAjaxSuccess.tribeEvents', obj.deinit );
	};

	/**
	 * Initialize filter button JS.
	 *
	 * @since  5.0.0
	 *
	 * @param  {Event}   event      event object for 'afterSetup.tribeEvents' event.
	 * @param  {integer} index      jQuery.each index param from 'afterSetup.tribeEvents' event.
	 * @param  {jQuery}  $container jQuery object of view container.
	 *
	 * @return {void}
	 */
	obj.init = function( event, index, $container ) {
		obj.initState( $container );
		obj.bindEvents( $container );
		$container.on( 'beforeAjaxSuccess.tribeEvents', { container: $container }, obj.deinit );
	};

	/**
	 * Handles the initialization of filter button when Document is ready.
	 *
	 * @since 5.0.0
	 *
	 * @return {void}
	 */
	obj.ready = function() {
		$document.on( 'afterSetup.tribeEvents', tribe.events.views.manager.selectors.container, obj.init );
	};

	// Configure on document ready.
	$( obj.ready );
} )( jQuery, tribe.filterBar.filterButton );
