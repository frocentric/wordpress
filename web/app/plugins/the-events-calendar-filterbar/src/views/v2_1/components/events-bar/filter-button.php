<?php
/**
 * View: Filter Button Component
 *
 * Override this template in your own theme by creating a file at:
 * [your-theme]/tribe/events-filterbar/v2_1/components/events-bar/filter-button.php
 *
 * See more documentation about our views templating system.
 *
 * @link http://evnt.is/1aiy
 *
 * @var string $breakpoint_pointer String we use as pointer to the current view we are setting up with breakpoints.
 * @var string $filterbar_state    Current state of the entire Filter Bar, either `open` or `closed`.
 *
 * @version 5.0.0
 *
 */

$button_classes = [ 'tribe-events-c-events-bar__filter-button' ];

if ( empty( $filterbar_state ) || 'closed' === $filterbar_state ) {
	$button_text   = __( 'Show filters', 'tribe-events-filter-view' );
	$aria_expanded = 'false';
} else {
	$button_classes[] = 'tribe-events-c-events-bar__filter-button--active';
	$button_text      = __( 'Hide filters', 'tribe-events-filter-view' );
	$aria_expanded    = 'true';
}
?>
<div class="tribe-events-c-events-bar__filter-button-container">
	<button
		<?php tribe_classes( $button_classes ); ?>
		aria-controls="tribe-filter-bar--<?php echo esc_attr( $breakpoint_pointer ); ?>"
		aria-expanded="<?php echo esc_attr( $aria_expanded ); ?>"
		data-js="tribe-events-filter-button"
	>
		<?php $this->template( 'components/icons/filter', [ 'classes' => [ 'tribe-events-c-events-bar__filter-button-icon' ] ] ); ?>
		<span class="tribe-events-c-events-bar__filter-button-text tribe-common-b2 tribe-common-a11y-visual-hide">
			<?php echo esc_html( $button_text ); ?>
		</span>
	</button>
</div>
