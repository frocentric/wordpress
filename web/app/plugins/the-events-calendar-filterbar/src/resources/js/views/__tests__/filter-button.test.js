describe( 'Filter Button', () => {
	beforeAll( () => {
		String.prototype.className = function() {
			if (
				(
					'string' !== typeof this &&
					! this instanceof String /* eslint-disable-line no-unsafe-negation */
				) ||
				'function' !== typeof this.replace
			) {
				return this;
			}

			return this.replace( '.', '' );
		};

		global.tribe = {
			filterBar: {
				filterBarState: {},
			},
		};
		require( '../filter-button' );
		tribe.events = {
			views: {
				accordion: {},
			},
		};
	} );

	afterAll( () => {
		delete String.prototype.className;
		delete global.tribe;
	} );

	beforeEach( () => {
		tribe.filterBar.filterBarState.closeFilterBar = jest.fn();
		tribe.filterBar.filterBarState.openFilterBar = jest.fn();
	} );

	describe( 'Selectors', () => {
		test( 'Should match snapshot', () => {
			const selectors = JSON.stringify( tribe.filterBar.filterButton.selectors );
			expect( selectors ).toMatchSnapshot();
		} );
	} );

	describe( 'Handle resize', () => {
		let $filterBar;
		let $container;

		beforeEach( () => {
			$filterBar = $( '<div></div>' );
			$container = $( '<div></div>' );
			$container.find = () => $filterBar;
		} );

		test( 'Should open filter bar on resize from mobile to desktop if layout is vertical', () => {
			// Setup test.
			$filterBar.data( 'tribeEventsState', { filterButtonDesktopInitialized: false } );
			$container.data( 'tribeEventsState', { isMobile: false } );
			$container.is = () => true;
			const event = {
				data: {
					container: $container,
				},
			};

			// Confirm initial state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();

			// Test.
			tribe.filterBar.filterButton.handleResize( event );

			// Confirm final state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();
			expect( tribe.filterBar.filterBarState.openFilterBar.mock.calls.length ).toBe( 1 );
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
		} );

		test( 'Should not open filter bar on resize from mobile to desktop if layout is not vertical', () => {
			// Setup test.
			$filterBar.data( 'tribeEventsState', { filterButtonDesktopInitialized: false } );
			$container.data( 'tribeEventsState', { isMobile: false } );
			$container.is = () => true;
			const event = {
				data: {
					container: $container,
				},
			};

			// Confirm initial state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();

			// Test.
			tribe.filterBar.filterButton.handleResize( event );

			// Confirm final state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();
			expect( tribe.filterBar.filterBarState.openFilterBar.mock.calls.length ).toBe( 1 );
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
		} );

		test( 'Should close filter bar on resize from desktop to mobile', () => {
			// Setup test.
			$filterBar.data( 'tribeEventsState', { filterButtonDesktopInitialized: true } );
			$container.data( 'tribeEventsState', { isMobile: true } );
			const event = {
				data: {
					container: $container,
				},
			};

			// Confirm initial state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();

			// Test.
			tribe.filterBar.filterButton.handleResize( event );

			// Confirm final state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();
			expect( tribe.filterBar.filterBarState.openFilterBar.mock.calls.length ).toBe( 0 );
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 1 );
		} );

		test( 'Should not close filter bar on resize from mobile to mobile', () => {
			// Setup test.
			$filterBar.data( 'tribeEventsState', { filterButtonDesktopInitialized: false } );
			$container.data( 'tribeEventsState', { isMobile: true } );
			const event = {
				data: {
					container: $container,
				},
			};

			// Confirm initial state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();

			// Test.
			tribe.filterBar.filterButton.handleResize( event );

			// Confirm final state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();
			expect( tribe.filterBar.filterBarState.openFilterBar.mock.calls.length ).toBe( 0 );
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
		} );

		test( 'Should not open filter bar on resize from desktop to desktop', () => {
			// Setup test.
			$filterBar.data( 'tribeEventsState', { filterButtonDesktopInitialized: true } );
			$container.data( 'tribeEventsState', { isMobile: false } );
			const event = {
				data: {
					container: $container,
				},
			};

			// Confirm initial state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();

			// Test.
			tribe.filterBar.filterButton.handleResize( event );

			// Confirm final state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();
			expect( tribe.filterBar.filterBarState.openFilterBar.mock.calls.length ).toBe( 0 );
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
		} );
	} );

	describe( 'Handle click', () => {
		let $container;

		beforeEach( () => {
			$container = $( '<div></div>' );
		} );

		test( 'Should return early if not mobile', () => {
			// Setup test.
			$container.data( 'tribeEventsState', { isMobile: false } );
			const event = {
				data: {
					container: $container,
				},
			};

			// Test.
			tribe.filterBar.filterButton.handleClick( event );

			// Confirm final state.
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
		} );

		test( 'Should not close filter bar if click target parent is filter bar', () => {
			// Setup test.
			$container.data( 'tribeEventsState', { isMobile: true } );
			const filterBar = `
				<div
					class="tribe-filter-bar tribe-filter-bar--horizontal"
					id="tribe-filter-bar--12345"
					data-js="tribe-filter-bar"
				>
					<form
						class="tribe-filter-bar__form"
						method="post"
						action=""
						aria-labelledby="tribe-filter-bar__form-heading--12345"
						aria-describedby="tribe-filter-bar__form-description--12345"
					>
					</form>
				</div>
			`;
			const $filterBar = $( filterBar );
			const event = {
				target: $filterBar.find( 'form' )[ 0 ],
				data: {
					container: $container,
				},
			};

			// Test.
			tribe.filterBar.filterButton.handleClick( event );

			// Confirm final state.
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
		} );

		test( 'Should not close filter bar if click target parent is filter button', () => {
			// Setup test.
			$container.data( 'tribeEventsState', { isMobile: true } );
			const filterButton = `
				<button
					class="tribe-events-c-events-bar__filter-button"
					aria-controls="tribe-filter-bar--12345"
					aria-expanded="false"
					data-js="tribe-events-accordion-trigger tribe-events-filter-button"
				>
				</button>
			`;
			const $filterButton = $( filterButton );
			const event = {
				target: $filterButton[ 0 ],
				data: {
					container: $container,
				},
			};

			// Test.
			tribe.filterBar.filterButton.handleClick( event );

			// Confirm final state.
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
		} );

		test( 'Should close filter bar if click target parent not filter bar or filter button', () => {
			// Setup test.
			$container.data( 'tribeEventsState', { isMobile: true } );
			const event = {
				target: $container[ 0 ],
				data: {
					container: $container,
				},
			};

			// Test.
			tribe.filterBar.filterButton.handleClick( event );

			// Confirm final state.
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 1 );
		} );
	} );

	describe( 'Handle action done click', () => {
		test( 'Should close filter bar', () => {
			// Setup test.
			const event = {
				data: {
					container: $(),
				},
			};

			// Test.
			tribe.filterBar.filterButton.handleActionDoneClick( event );

			// Confirm final states.
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 1 );
		} );
	} );

	describe( 'Handle filter button click', () => {
		let $container;
		let $filterButtonText;
		let $filterBar;

		beforeEach( () => {
			tribe.filterBar.filterBarSlider = {
				handleResize: jest.fn(),
			};
			$container = $( '<div></div>' );
			$filterButtonText = $( '<span></span>' );
			$filterBar = $( '<div></div>' );
			$container.find = () => $filterBar;
			$filterButtonText.text = jest.fn();
			$filterBar.toggleClass = jest.fn();
		} );

		afterEach( () => {
			delete global.tribe.filterBar.filterBarSlider;
		} );

		test( 'Should close filter bar on click', () => {
			// Setup test.
			const $filterButton = $( '<button></button>' );
			$container.data = () => ( { isMobile: false } );
			$filterButton.addClass( tribe.filterBar.filterButton.selectors.filterButtonActive.className() );
			$filterButton.find = () => $filterButtonText;
			$filterButton.toggleClass = jest.fn();
			const event = {
				data: {
					target: $filterButton,
					actionDone: $(),
					container: $container,
				},
			};

			// Test.
			tribe.filterBar.filterButton.handleFilterButtonClick( event );

			// Confirm final states.
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 1 );
			expect( tribe.filterBar.filterBarState.openFilterBar.mock.calls.length ).toBe( 0 );
			expect( tribe.filterBar.filterBarSlider.handleResize.mock.calls.length ).toBe( 1 );
		} );

		test( 'Should open filter bar on click', () => {
			// Setup test.
			const $filterButton = $( '<button></button>' );
			$container.data = () => ( { isMobile: false } );
			$filterButton.find = () => $filterButtonText;
			$filterButton.toggleClass = jest.fn();
			const event = {
				data: {
					target: $filterButton,
					actionDone: $(),
					container: $container,
				},
			};

			// Test.
			tribe.filterBar.filterButton.handleFilterButtonClick( event );

			// Confirm final states.
			expect( tribe.filterBar.filterBarState.closeFilterBar.mock.calls.length ).toBe( 0 );
			expect( tribe.filterBar.filterBarState.openFilterBar.mock.calls.length ).toBe( 1 );
			expect( tribe.filterBar.filterBarSlider.handleResize.mock.calls.length ).toBe( 1 );
		} );
	} );

	describe( 'Initialize state', () => {
		test( 'Should initialize state if filter bar is vertical', () => {
			// Setup test.
			const $container = $( '<div></div>' );
			const $filterBar = $( '<div></div>' );
			$container.find = () => $filterBar;
			$container.data( 'tribeEventsState', { isMobile: true } );

			// Test.
			tribe.filterBar.filterButton.initState( $container );

			// Confirm final state.
			expect( $filterBar.data( 'tribeEventsState' ) ).toMatchSnapshot();
		} );
	} );
} );
