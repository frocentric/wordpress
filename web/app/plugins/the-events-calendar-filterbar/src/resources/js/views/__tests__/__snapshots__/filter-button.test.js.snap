// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter Button Handle resize Should close filter bar on resize from desktop to mobile 1`] = `
Object {
  "filterButtonDesktopInitialized": true,
}
`;

exports[`Filter Button Handle resize Should close filter bar on resize from desktop to mobile 2`] = `
Object {
  "filterButtonDesktopInitialized": false,
}
`;

exports[`Filter Button Handle resize Should not close filter bar on resize from mobile to mobile 1`] = `
Object {
  "filterButtonDesktopInitialized": false,
}
`;

exports[`Filter Button Handle resize Should not close filter bar on resize from mobile to mobile 2`] = `
Object {
  "filterButtonDesktopInitialized": false,
}
`;

exports[`Filter Button Handle resize Should not open filter bar on resize from desktop to desktop 1`] = `
Object {
  "filterButtonDesktopInitialized": true,
}
`;

exports[`Filter Button Handle resize Should not open filter bar on resize from desktop to desktop 2`] = `
Object {
  "filterButtonDesktopInitialized": true,
}
`;

exports[`Filter Button Handle resize Should not open filter bar on resize from mobile to desktop if layout is not vertical 1`] = `
Object {
  "filterButtonDesktopInitialized": false,
}
`;

exports[`Filter Button Handle resize Should not open filter bar on resize from mobile to desktop if layout is not vertical 2`] = `
Object {
  "filterButtonDesktopInitialized": true,
}
`;

exports[`Filter Button Handle resize Should open filter bar on resize from mobile to desktop if layout is vertical 1`] = `
Object {
  "filterButtonDesktopInitialized": false,
}
`;

exports[`Filter Button Handle resize Should open filter bar on resize from mobile to desktop if layout is vertical 2`] = `
Object {
  "filterButtonDesktopInitialized": true,
}
`;

exports[`Filter Button Initialize state Should initialize state if filter bar is vertical 1`] = `
Object {
  "filterButtonDesktopInitialized": false,
}
`;

exports[`Filter Button Selectors Should match snapshot 1`] = `"{\\"actionDone\\":\\"[data-js=\\\\\\"tribe-filter-bar__action-done\\\\\\"]\\",\\"filterBarVertical\\":\\".tribe-events--filter-bar-vertical\\",\\"filterBar\\":\\"[data-js~=\\\\\\"tribe-filter-bar\\\\\\"]\\",\\"filterButton\\":\\"[data-js~=\\\\\\"tribe-events-filter-button\\\\\\"]\\",\\"filterButtonActive\\":\\".tribe-events-c-events-bar__filter-button--active\\",\\"filtersSliderContainer\\":\\"[data-js=\\\\\\"tribe-filter-bar-filters-slider-container\\\\\\"]\\",\\"select2ChoiceRemove\\":\\".select2-selection__choice__remove\\"}"`;
